docs/usage/upgrade-best-practices.md

```diff
@@ -0,0 +1,270 @@
+- read the previous changelogs
+
+#### You'll be ready for CVE patches
+
+Thirdly, you'll be ready when a upstream package releases a patch for a critical CVE.
+If you're current, you can review and merge Renovate's PR quickly.
+
+When you're behind on updates, you'll have a bad time, because you must read _more_ changelogs and make _more_ changes before you can merge the critical patch.
+
+#### You'll look for ways to automate the updates
+
+Finally, when you're updating often, you'll start looking for ways to automate the updates.
+You may start to [`automerge`](./configuration-options.md#automerge) development dependencies like Prettier, or ESLint when the linter passes.
+Or you may decide to automerge any `patch` type upgrades, by using the [`default:automergePatch`](https://docs.renovatebot.com/presets-default/#automergepatch) preset.
+
+You may also start using [GitHub's pull request merge queues](./key-concepts/automerge.md#github-merge-queue) to speed up the merge process.
+Renovate does not support GitLab's Merge Trains, see [issue #5573](https://github.com/renovatebot/renovate/issues/5573).
+
+## Starting from a new project
+
+Let's assume you start a new project.
+You created a new Git repository, installed the latest frameworks, libraries and development tools.
+After pushing the initial commit, you should enable and onboard Renovate.
+
+Now you'll have to stay on the "update often" train.
+
+## Project with one year old dependencies
+
+If you have a project that's a year behind on dependencies, you'll need to do some work.
+Let's assume that most dependencies need a `patch` or `minor` update, and at _least_ one dependency needs a `major` update.
+We recommend you start small, and do the easier updates first.
+
+Start with the `patch` and `minor` updates.
+Read the changelogs for your updates.
+You may have to make small changes to get things working again.
+
+When you have the latest `patch` and `minor` versions, you are ready for `major` updates.
+Start with `major` version updates for tools like Prettier or ESLint.
+
+Then work on `major` updates for your framework or library.
+Take your time, read the changelogs, and make the necessary changes.
+Let multiple team members review your work before merging, it's easy to miss something.
+
+Finally, update your development tools.
+
+Now that you're up to date, it's important to start thinking about how to make updating a regular thing.
+
+## Project with five year old dependencies
+
+Let's assume your Dependency Dashboard lists more than 50 updates, and you have a few `major` version updates pending.
+If your project is this badly behind on updates, you have two problems:
+
+- Updating your dependencies
+- Improving your update process
+
+### Focus on critical updates first
+
+Let's fix the easier problem first: getting back up to date.
+First update any dependencies that have critical updates for CVEs or other security related improvements.
```

Comment:
We could reference our vulnerabilitiy options here

Reply:
```suggestion
First update any dependencies that have critical updates for CVEs or other security related improvements.

Follow the steps listed in the [`vulnerabilityAlerts`](https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts) docs to make sure Renovate is reading the GitHub Vulnerability Alerts.
```

Is this what you mean? I'm not quite sure if you _must_ have `vulnerabilityAlerts` set up to get CVE fixes, or if this is just about customizing the CVE fixes you would get anyways...

Reply:
It has to be set up and only works for Github. 

What I reffered to was `osvVulnerabilities` too